### DISCUSSION TITLE
Call shortcode from custom event field

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#419] [General]
Is it possible to call a shortcode form an event custom field? I've put together some code to insert breadcrumbs on my website, which I'm inserting using the Shortcoder plugin, and also want to use this on event pages. So I created a custom field for events to hold the breadcrumb shortcode with custom text & link for each event page., and inserted the shortcode like this:
`[sc name="sc-breadcrumbs"]<li>Current page</li>[/sc]` 

Then on the event page I insert this code:
`[eme_if tag='#_FIELD{BREADCRUMB}' is_empty=0]#_FIELD{BREADCRUMB}[/eme_if]`

This doesn't insert anything onto the page. So I tried putting the shortcoder [sc] tag in the event template and put the breadcrumb `<li>` code in the custom field. This works, but I'm just wondering if it's possible to run shortcodes from custom event fields using the custom field placeholder? It would just be cleaner to keep the shortcode and its parameters together.

**Comment by liedekef** [comment]
expanding #_FIELD{BREADCRUMB} in the eme_if-tag part would result in:
`[eme_if tag=' [sc name="sc-breadcrumbs"]<li>Current page</li>[/sc]' is_empty=0]`
So a shortcode inside another shortcode-element, which is not supported by WP

**Comment by TimB111** [comment]
I see... Ok no problem, thanks for explaining.
